staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -366,6 +371,10 @@ type CustomResourceDefinitionStatus struct {
 	// versions from this list.
 	// None of the versions in this list can be removed from the spec.Versions field.
 	StoredVersions []string
+
+	// The generation observed by the CRD controller. Gated on the alpha
+	// CRDObservedGenerationTracking feature gate.
```

Comment:
```suggestion
	// The generation observed by the CRD controller.
	// +featureGate=CRDObservedGenerationTracking
	// +optional
```

Reply:
Ah, didn't know there was a flag for that.